File: concrete/src/main/java/pcb/uwu/unit/base/KiloGramUnit.java
package pcb.uwu.unit.base;

import pcb.uwu.units.base.MassUnit;

import java.util.function.Function;

public class KiloGramUnit extends MassUnit {

	public static final KiloGramUnit KILOGRAM = new KiloGramUnit();

	private KiloGramUnit() {
		super("Kg", "kilogram", "kilograms",
				Function.identity(),
				Function.identity());
	}
}

File: concrete/src/main/java/pcb/uwu/unit/base/KiloGramUnit.kt
package pcb.uwu.unit.base

import java.util.function.Function.identity

object KiloGramUnit
    : MassUnit(symbol = "Kg",
               singularName = "kilogram",
               pluralName = "kilograms",
               translationToCanonical = identity(),
               translationFromCanonical = identity())
